evernode-governor-hook/src/governor.c

```diff
@@ -851,6 +829,39 @@ int64_t hook(uint32_t reserved)
                 PERMIT_M("Evernode: Piloted mode candidate status changed.", vote_status);
             }
         }
+
+        if (redirect_op_type == OP_CHANGE_CONFIGURATION)
+        {
+            SET_UINT_STATE_VALUE(DEF_MOMENT_SIZE, CONF_MOMENT_SIZE, "Evernode: Could not initialize state for moment size.");
+            SET_UINT_STATE_VALUE(DEF_MINT_LIMIT, CONF_MINT_LIMIT, "Evernode: Could not initialize state for mint limit.");
+            SET_UINT_STATE_VALUE(DEF_FIXED_REG_FEE, CONF_FIXED_REG_FEE, "Evernode: Could not initialize state for fixed reg fee.");
+            
+            // <epoch_count(uint8_t)><first_epoch_reward_quota(uint32_t)><epoch_reward_amount(uint32_t)><reward_start_moment(uint32_t)><accumulated_reward_frequency(uint16_t)>
+            uint8_t reward_configuration[REWARD_CONFIGURATION_VAL_SIZE];
+            reward_configuration[EPOCH_COUNT_OFFSET] = DEF_EPOCH_COUNT;
+            UINT32_TO_BUF_LE(&reward_configuration[FIRST_EPOCH_REWARD_QUOTA_OFFSET], DEF_FIRST_EPOCH_REWARD_QUOTA);
+            UINT32_TO_BUF_LE(&reward_configuration[REWARD_START_MOMENT_OFFSET], DEF_REWARD_START_MOMENT);
+            UINT16_TO_BUF_LE(&reward_configuration[ACCUMULATED_REWARD_FREQUENCY_OFFSET], DEF_ACCUMULATED_REWARD_FREQUENCY);
```

Comment:
I think we should only allow to change the ACCUMULATED_REWARD_FREQUENCY in hook update